From 9e5fe604ee901f5cc96ebbc7ecae6d7514de7a0c Mon Sep 17 00:00:00 2001
From: Pan Xiuli <xiuli.pan@linux.intel.com>
Date: Wed, 21 Mar 2018 16:38:53 +0800
Subject: [PATCH 0876/1040] ASoC: SOF: export snd_sof_trace_notify_for_error

Need to use this functionn in other place, export it.

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>
---
 sound/soc/sof/trace.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/sof/trace.c b/sound/soc/sof/trace.c
index 7e22ee4b9301..dc452ad61093 100644
--- a/sound/soc/sof/trace.c
+++ b/sound/soc/sof/trace.c
@@ -265,6 +265,7 @@ void snd_sof_trace_notify_for_error(struct snd_sof_dev *sdev)
 		wake_up(&sdev->trace_sleep);
 	}
 }
+EXPORT_SYMBOL(snd_sof_trace_notify_for_error);
 
 void snd_sof_release_trace(struct snd_sof_dev *sdev)
 {
-- 
2.17.1

